frontend/public/components/deployment-config.tsx

```diff
@@ -47,9 +49,15 @@ const rollout = (dc: K8sResourceKind): Promise<K8sResourceKind> => {
   return k8sCreate(DeploymentConfigModel, req, opts);
 };
 
+const determineReplicationControllerName = (dc: K8sResourceKind): string => {
```

Comment:
we are doing the same thing in [origin](https://github.com/openshift/origin/blob/master/pkg/apps/util/util.go#L305)